cpp/include/cudf/io/json.hpp

```diff
@@ -225,6 +227,13 @@ class json_reader_options {
    */
   bool is_enabled_lines() const { return _lines; }
 
+  /**
+   * @brief Whether to read the mixed types as string column.
+   *
+   * @return `true` if reading the mixed types as string column
```

Comment:
I'd propose this wording. This is more of a "parsing" choice than a "reading" choice, I think.
```suggestion
   * @brief Whether to parse mixed types as a string column.
   *
   * @return `true` if mixed types are parsed as a string column
```